src/commands/sourcemaps/upload.ts

```diff
@@ -228,11 +229,15 @@ export class UploadCommand extends Command {
     return getRequestBuilder({
       apiKey: this.config.apiKey!,
       baseUrl: getBaseIntakeUrl(),
+      disableEnvironmentVariables: true,
```

Comment:
Is there a reason not to enable this setting by default for all commands @m-rousse ?